src/index.tsx

```diff
@@ -2,6 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './components/App';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
I also tend to group import statements - like all external dependencies first then internal files.  (Technically I also alphabetize em in their groups but maybe that's overkill ;) ).  Food for thought, no need to change code.

Reply:
Good idea. Done!